spec/controllers/target_controller_spec.rb

```diff
@@ -4,10 +4,22 @@
   let(:target) { build :target }
 
   describe 'POST create' do
-    it 'creates target' do
-      expect do
-        post :create, params: { target: target.attributes }
-      end.to change(Target, :count).by(1)
+    context 'when the user is logged in' do
+      login_user
+
```

Comment:
You could also test:
- what happens when invalid params are sent 
- request responses
- The data of the created target, was the target created with the expected data?